app/src/main/java/com/vestrel00/daggerbutterknifemvp/AppComponent.java

```diff
@@ -20,28 +20,25 @@
 
 import javax.inject.Singleton;
 
-import dagger.BindsInstance;
 import dagger.Component;
+import dagger.android.AndroidInjector;
+import dagger.android.support.AndroidSupportInjectionModule;
 
 /**
  * Injects application dependencies.
  */
 @Singleton
-@Component(modules = AppModule.class)
-interface AppComponent {
-    void inject(App app);
-
+@Component(modules = {
+        AndroidSupportInjectionModule.class,
```

Comment:
I created setup that uses *dagger.android.support* with AppCompatActivity and support Fragments. Take a look at #49.